src/utils/SaveDataMigrationUtils.ts

```diff
@@ -550,6 +552,16 @@ Error: ${e}`,
     if (found) Terminal.error("Filenames with whitespace found and corrected, see console for details.");
   }
   if (ver < 44) {
+    try {
+      /**
+       * Backup pre-v3 save data. We must use the data in IndexedDB instead of calling saveObject.getSaveData().
+       * getSaveData() returns data in v3 format, so the exported data will not be importable in pre-v3.
+       */
+      const saveData = await load();
+      downloadContentAsFile(saveData, `bitburnerSave_backup_${Math.round(Date.now() / 1000)}.json.gz`);
```

Comment:
Let's put v2.8.1 in here. Also, can we base the timestamp on the lastUpdated of the save itself, rather than current time? That way, if you end up triggering this multiple times (let's say the game is crashing on startup), it will generate copies of the same file instead of new ones.